db/collection.go

```diff
@@ -341,13 +431,35 @@ func (t *Txn) Delete(ids ...core.InstanceID) error {
 
 // Has returns true if all IDs exists in the collection, false otherwise.
 func (t *Txn) Has(ids ...core.InstanceID) (bool, error) {
+	if err := t.collection.db.connector.Validate(t.token, true); err != nil {
+		return false, err
+	}
+	pk, err := t.token.PubKey()
```

Comment:
I see now why the clarification in the exported comment that this doesn't check is verified. 
The previous `Validate` is solving that first.